pkg/config/default_windows_test.go

```diff
@@ -0,0 +1,77 @@
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetDefaultMachineVolumes(t *testing.T) {
+	t.Parallel()
+	for _, tc := range []struct {
+		envs   [][]string
+		output []string
+	}{
+		{
+			envs: [][]string{
+				{"USERPROFILE", "C:\\Users\\test"},
+			},
+			output: []string{
+				"C:\\Users\\test:/Users/test",
+				"C:\\:/mnt/c",
+			},
+		},
+		{
+			envs: [][]string{
+				{"USERPROFILE", "C:\\Users\\test\\AppData\\Local\\Temp\\podman_test123456789"},
+			},
+			output: []string{
+				"C:\\Users\\test\\AppData\\Local\\Temp\\podman_test123456789:/Users/test/AppData/Local/Temp/podman_test123456789",
+				"C:\\:/mnt/c",
+			},
+		},
+		{
+			envs: [][]string{
+				{"USERPROFILE", "D:\\Users\\test"},
+			},
+			output: []string{
+				"D:\\Users\\test:/Users/test",
+				"D:\\:/mnt/d",
+			},
+		},
+		{
+			envs: [][]string{
+				{"USERPROFILE", "c:\\users\\test"},
+			},
+			output: []string{
+				"c:\\users\\test:/users/test",
+				"c:\\:/mnt/c",
+			},
+		},
+		{
+			envs: [][]string{
+				{"USERPROFILE", "C:/Users/test"},
+			},
+			output: []string{
+				"C:/Users/test:/Users/test",
+				"C:\\:/mnt/c",
+			},
+		},
+		{
+			envs: [][]string{
+				{"USERPROFILE", "C:\\Users\\test\\"},
+			},
+			output: []string{
+				"C:\\Users\\test\\:/Users/test/",
+				"C:\\:/mnt/c",
+			},
+		},
+	} {
+		for _, env := range tc.envs {
+			os.Setenv(env[0], env[1])
```

Comment:
use t.Setenv() then go should rightfully error out and tell you using setenv with parallel is not supported.